_includes/addons/opengraph.html

```diff
@@ -0,0 +1,7 @@
+<meta property="og:title" content="{{ site.title }}" />
+<meta property="og:site_name" content="{{ site.title }}" />
+<meta property="og:url" content="{{ site.url }}" />
+<meta property="og:description" content="{{  site.description | truncate: 200 }}" />
+<meta name="og:image" content="{{ site.url }}{{ site.logo }}" />
```

Comment:
FYI, you can make another commit to your `analytics` branch and github will update this pull request with your latest code